target-app/Onboarding/Views/SignUpViewController.swift

```diff
@@ -0,0 +1,224 @@
+//
+//  SignUpViewController.swift
+//  target-app
+//
+//  Created by Andres Leonel Torrico Cossio on 26/08/2022.
+//
+
+import UIKit
+
+class SignUpViewController: UIViewController {
+
+    private lazy var overlayImageView = UIImageView()
+    
+    private lazy var titleLabel = UILabel(style: .primary(text: "signin_title".localized))
```

Comment:
**FYI comment:** 
 This is  a good approach but maybe when your'e sure that the variable is going to be loaded the first the screen appear (and if you don't need to change it in the future) you can try to using just `private let` .ie: 

 `private let titleLabel = UILabel(style: .primary(text: "signin_title".localized))`
 
 The lazy var is very cool when you don't need to load that element immediately when the view is loading, because it is loaded just when we need it

Reply:
thanks! I'll fix it in next PR's